### PR TITLE
Move defaulting funcs from internal to v1

### PR DESCRIPTION
**Pull request by feihujiang**
Fix todo in api/conversion.go

### CODE REVIEW

File: pkg/api/serialization_test.go [hunk 1]
```diff
@@ -128,7 +128,8 @@ func TestList(t *testing.T) {
 }
 
 var nonRoundTrippableTypes = sets.NewString()
-var nonInternalRoundTrippableTypes = sets.NewString("List", "ListOptions", "PodExecOptions", "PodAttachOptions")
+
```
Comment (lavalamp) [inline pkg/api/serialization_test.go]: nit: extra blank line (and all three of these things could be in one `var ( ... )` block...)

**Comment by lavalamp** [comment]
LGTM, not blocking on nit

**Comment by feihujiang** [comment]
@k8s-bot unit test please

**Comment by feihujiang** [comment]
@k8s-bot test this please

**Comment by feihujiang** [comment]
@k8s-bot unit test this please

**Pull request merged**